common/common/src/main/java/io/helidon/common/FeatureCatalog.java

```diff
@@ -150,7 +150,7 @@
                     .name("Websocket")
                     .description("Jakarta Websocket implementation")
                     .path("WebServer", "Websocket")
-                    .nativeSupported(false));
+                    .nativeSupported(true));
```

Comment:
Need to add nativeDescription that declares that only server is supported (unless this is feature for server only). If for server only, make sure we have a client feature as well that declares it is not supported (if possible).

Reply:
fixed